src/main/kotlin/by/architecture_map/belarus/service/impl/SearchServiceImpl.kt

```diff
@@ -50,4 +51,24 @@ class SearchServiceImpl(
 
         return searchHits.map { it.content }.toList()
     }
+
+    override fun searchArticle(request: String): List<Article> {
```

Comment:
it is better to split search service per entity. or you can move it to the corresponding entity services